Sources/Core/AWSSDKHTTPAuth/Sources/AWSSDKHTTPAuth/SigV4AuthScheme.swift

```diff
@@ -15,9 +15,14 @@ import struct Smithy.Attributes
 public struct SigV4AuthScheme: AuthScheme {
     public let schemeID: String = "aws.auth#sigv4"
     public let signer: Signer = AWSSigV4Signer()
+    public var requestUnsignedBody: Bool?
```

Comment:
The possible values for `Bool?` are `true`, `false`, and `nil`.

`true` means unsigned body is requested.
`nil` means unsigned body was not specified in the auth scheme resolver.

So what is `false`?  I can certainly create a auth scheme resolver with `requestUnsignedBody: false`.  Would that ever be used?  Is it undefined?  (If it doesn't, we should consider a different data type than `Bool?` for this setting.)